src/applications/caregivers/components/AdditionalInfo/index.jsx

```diff
@@ -275,3 +275,30 @@ export const RepresentativeAdditionalInfo = () => {
     </div>
   );
 };
+
+export const RepresentativeDocumentUploadDescription = () => {
+  return (
+    <section>
+      <h3 style={{ padding: 0, marginBottom: '1.3em' }}>
+        Upload your supporting documentation
+      </h3>
+
+      <p>
+        You will first need to scan a copy of your document onto the same device
+        that you are submitting your online application (i.e. computer or mobile
```

Comment:
Not to be a grammar cop, but this should be `e.g.` not `i.e.`... and how does one scan a document into a mobile phone?

Reply:
Roger that, I will pass this to @Dene-human our new designer on Caregiver for feedback, this is not the final copy so will get it in the next run.

Reply:
I certainly agree @Mottie. This is something that is directly from the project team that we need in our research study, but is definitely not final content. Thank you for checking!